tests/edgeless/note.spec.ts

```diff
@@ -143,23 +145,22 @@ test('add empty Note', async ({ page }) => {
   await switchEditorMode(page);
   await zoomResetByKeyboard(page);
   await setEdgelessTool(page, 'note');
-
+  await page.pause();
```

Comment:
Don't commit this. And please also remove all the `page.pause()` in the repository - they all git blame to you 🫤.

<img width="980" alt="Screenshot 2023-10-23 at 18 38 53" src="https://github.com/toeverything/blocksuite/assets/7312949/6ccdbdd8-2de2-4ade-b7b3-964db96dafb4">